src/ui/src/builder/BuilderEmbeddedCodeEditor.vue

```diff
@@ -108,6 +114,10 @@ onUnmounted(() => {
 	min-height: 300px;
 }
 
+.BuilderEmbeddedCodeEditor--half-screen {
```

Comment:
Nitpick: Usually, we use `camelCase` for modifiers.
```suggestion
.BuilderEmbeddedCodeEditor--halfScreen {
```